doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -43,69 +43,366 @@ On your computer, one of the following operating systems:
+
+      *serial_number*
+      adac_auth_version     1.0
+      vendor_id             Nordic VLSI ASA
+      soc_class             0x00005420
+      soc_id                [e6, 6f, 21, b6, dc, be, 11, ee, e5, 03, 6f, fe, 4d, 7b, 2e, 07]
+      hw_permissions_fixed  [00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      hw_permissions_mask   [01, 00, 00, 00, 87, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      psa_lifecycle         LIFECYCLE_EMPTY (0x1000)
+      sda_id                0x01
+      secrom_revision       0xad3b3cd0
+      sysrom_revision       0xebc8f190
+      token_formats         [TokenAdac]
+      cert_formats          [CertAdac]
+      cryptosystems         [Ed25519Sha512]
+      Additional TLVs:
+      TargetIdentity: [ff, ff, ff, ff, ff, ff, ff, ff]
+
+#. If the lifecycle state (``psa_lifecycle``) shown is ``RoT`` (``LIFECYCLE_ROT (0x2000)``), no LCS transition is required.
+   If the lifecycle state (``psa_lifecycle``) shown is not ``RoT`` (``LIFECYCLE_EMPTY (0x1000)`` means the LCS is set to ``EMPTY``), set it to Root of Trust using the following command::
+
+      nrfutil device x-adac-lcs-change --life-cycle rot --serial-number <serial_number>
+
+#. Verify again the current lifecycle state of the nRF54H20::
+
+      nrfutil device x-adac-discovery --serial-number <serial_number>
+
+   The output will look similar to the following::
+
+      *serial_number*
+      adac_auth_version     1.0
+      vendor_id             Nordic VLSI ASA
+      soc_class             0x00005420
+      soc_id                [e6, 6f, 21, b6, dc, be, 11, ee, e5, 03, 6f, fe, 4d, 7b, 2e, 07]
+      hw_permissions_fixed  [00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      hw_permissions_mask   [01, 00, 00, 00, 87, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      psa_lifecycle         LIFECYCLE_ROT (0x2000)
+      sda_id                0x01
+      secrom_revision       0xad3b3cd0
+      sysrom_revision       0xebc8f190
+      token_formats         [TokenAdac]
+      cert_formats          [CertAdac]
+      cryptosystems         [Ed25519Sha512]
+      Additional TLVs:
+      TargetIdentity: [ff, ff, ff, ff, ff, ff, ff, ff]
+
+   The lifecycle state (``psa_lifecycle``) is now correctly set to *Root of Trust* (``LIFECYCLE_ROT (0x2000)``)
+
+#. After the LCS transition, reset the device::
+
+      nrfutil device reset --reset-kind RESET_PIN --serial-number <serial_number>
+
 .. _ug_nrf54h20_gs_sample:
 
 Programming the sample
 **********************
 
-The :ref:`multicore_hello_world` sample is a multicore sample running on both the Application core (``cpuapp``) and the Peripheral Processor (PPR, ``cpuppr``).
-It uses the ``nrf54h20dk/nrf54h20/cpuapp`` build target.
+The :ref:`multicore_hello_world` sample is a multicore sample running on both the Application Core (``cpuapp``) and the Peripheral Processor (PPR, ``cpuppr``).
+It uses the ``nrf54h20dk_nrf54h20_cpuapp`` build target.
```

Comment:
Replace all "build target" with "board target", as "build target" is an incorrect term.
```suggestion
It uses the ``nrf54h20dk_nrf54h20_cpuapp`` board target.
```